be/engine.c

```diff
@@ -822,7 +823,7 @@ M0_INTERNAL int m0_be_engine_start(struct m0_be_engine *en)
 	if (en->eng_cfg->bec_wait_for_recovery) {
 		m0_semaphore_down(&en->eng_recovery_wait_sem);
 		recovery_time = m0_time_now() - recovery_time;
-		M0_LOG(M0_INFO, "BE recovery execution time: %lu",
+		M0_LOG(M0_INFO, "BE recovery execution time: %"PRId64,
```

Comment:
PRIu64

Reply:
$PRIu64 for %lu and PRId64 for %ld, do same at other places.

Reply:
fixed in 915ed2b